src/python/WMCore/FwkJobReport/Report.py

```diff
@@ -1323,6 +1323,19 @@ def setAcquisitionProcessing(self, acquisitionEra, processingVer, processingStr=
 
         return
 
+    def updateSubprocessInfo(self, cpuTime, startTime, endTime):
+        """
+        Add process information to WMCore FJR
+        :param cpuTime: cpu system time
+        :param startTime: start time of subprocess
+        :param endTime: endtime time of subprocess
+        """
+        self.report.section_('WMCMSSWSubprocess')
+        self.report.WMCMSSWSubprocess.startTime = startTime
+        self.report.WMCMSSWSubprocess.endTime = endTime
+        self.report.WMCMSSWSubprocess.wallClockTime = endTime - startTime
+        self.report.WMCMSSWSubprocess.systemTime = cpuTime
```

Comment:
Repeating my question from https://github.com/dmwm/WMCore/issues/11660#issuecomment-1661081899, why record only system time? Vast majority of `cmsRun`'s CPU time should be spent in user mode.